src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -143,6 +144,25 @@ case class LocalBackend(actorSystem: ActorSystem) extends Backend with SharedFil
     s"docker run --rm -v ${backendCall.callRootPath.toAbsolutePath}:$callPath -i $image"
   }
 
+  /**
+    * If using Mac OS X with Docker Machine, by default Cromwell can only use the -v flag to 'docker run'
+    * if running from somewhere within the user's home directory.  If not, then -v will mount to the
+    * virtual machine that Docker Machine creates.  This ends up bubbling up to Cromwell when it can't find
+    * an RC file in the call's output directory (because it's actually on the virtual machine).
+    *
+    * This function will at least detect this case and log some WARN messages.
+    */
+  private def detectDockerMachinePossibleMisusage(backendCall: BackendCall, logger: WorkflowLogger) = {
```

Comment:
The tricky thing about this is that it's possible that the user configured virtualbox to allow for running cromwell outside of `~`.  If we move the check to startup, then we have to have a way to query virtualbox to see if it exposes the directory we're running Cromwell under.  That seemed kinda hard to me, but maybe it's not?